packages/tools/devtools/devtools-browser-extension/jest.config.cjs

```diff
@@ -4,11 +4,17 @@
  */
 
 // Get the test port from the global map and set it in env for this test
-const testTools = require("@fluidframework/test-tools");
-const { name } = require("./package.json");
-
-const mappedPort = testTools.getTestPort(name);
-process.env["PORT"] = mappedPort;
+let mappedPort = 9000; // Defaul port test-tools would provide if no mapping exists
```

Comment:
Here and elsewhere:

```suggestion
let mappedPort = 9000; // Default port test-tools would provide if no mapping exists
```

Reply:
Not sure. I did try, but it cannot be in a separate package because when we load the jest config files due to policy checks, the repo isn't built, so the imports would fail. Making it a simple cjs file at the root and importing by path, maybe works?

Reply:
> Making it a simple cjs file at the root and importing by path, maybe works?

That's what I was thinking.